src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtils.java

```diff
@@ -1060,7 +1072,7 @@ protected static GenotypesContext updateGenotypesWithMappedAlleles(final Genotyp
      * <DEL>/C => <DEL>/C
      * @param alleles a non-null list of alleles that we want to clip
      * @return the number of bases from the end of the alleles where we can safely clip or
-     *   -1 if the clipping method isnt applicable. So, if the result is 1, then we can remove
+     *   -1 if the clipping method isn't applicable. So, if the result is 1, then we can remove
```

Comment:
We no longer have the special -1 case thanks to this PR.

Reply:
Fixed that.